fabfile.py

```diff
@@ -165,6 +180,198 @@ def listdir(path):
+    return field in FIELD_TYPE['choices']
+
+
+@task
+def check_dns_entries(data):
+    """Checks if the inputs in your dns_entries.json file are valid"""
+    entries_types = []
+    for domain in data:
+        print domain
+        print data[domain]
+        for base in data[domain]:
+            print base
+            entries_types.append(base["type"])
+    for item in entries_types:
+        print item
+
+    if all(check_field_type_exists(item) for item in entries_types):
+        return True
+    else:
+        for item in entries_types:
+            if not check_field_type_exists(item):
+                print("The field type {item} is not valid".format(item=item))
+                return False
+
+
+@task
+def check_domains(data):
+    """Checks if the domains you provided in the dns_entries.json are valid"""
+    domain_list = get_domain_list()
+    if all(domain in domain_list for domain in data):
+        return True
+    else:
+        for domain in data:
+            if domain not in domain_list:
+                print("The domain {domain} is not in the domain list".format(domain=domain))
+                return False
+
+
+@task
+def get_dns_entries_from_file():
+    data = json.load(open('configs/bluenove-server-configs/dns_entries.json'))
+    print json.dumps(data, indent=4)
+    return data
+
+
+@task
+def get_domain_list():
+    client = create_ovh_client()
+    result = client.get('/domain')
+    print json.dumps(result, indent=4)
+    return result
+
+
+@task
+def create_new_dns_record():
+    data = get_dns_entries_from_file()
+    if check_dns_entries(data) and check_domains(data):
+        client = create_ovh_client()
+        for domain in data:
+            for entry in data[domain]:
+                client.post('/domain/zone/{domain}/record'.format(domain=domain), fieldType=entry["type"], target=entry["value"])
+
+
+@task
+def create_new_dns_record_cmd_line_arguments(domain_entry, field_type_entry, target_entry):
+    client = create_ovh_client()
+    if domain_entry not in get_domain_list():
+        print("The domain {domain_entry} is not a valid domain".format(domain_entry=domain_entry))
+        print "The valid domains are:"
+        for domain in get_domain_list():
+            print domain
+    elif not check_field_type_exists(field_type_entry):
+        print("The field {field_type_entry} is not a valid field type".format(field_type_entry=field_type_entry))
+        print "Valid field types are A, AAAA, CNAME, DKIM, LOC, MX, NAPTR, NS, PTR, SPF, SRV, SSHFP, TXT"
+    else:
+        print("Creating new DNS record for the domain {domain_entry} with field type {field_type_entry} with target {target_entry}".format(
+            domain_entry=domain_entry, field_type_entry=field_type_entry, target_entry=target_entry))
+        client.post('/domain/zone/{domain_entry}/record'.format(domain_entry=domain_entry), fieldType=field_type_entry, target=target_entry)
```

Comment:
For `POST` operations, you should signal success or error and display the error. So, take the response of this `post`, check for success, return in `green` a success message, or in `red`, the failure message and return the exception, which would make `fabric` quit.